LR_Tourist/DA/Services/Repository/IRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DA.Services.Repository
+{
+    public interface IRepository<T>
+        where T : class
+    {
+        IEnumerable<T> GetAll();
```

Comment:
This service should be awaitable. Refactor all methods.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/

Reply:
fixed